src/relayer/Relayer.ts

```diff
@@ -367,6 +372,7 @@ export class Relayer {
     version: number,
     deposit: Deposit,
     symbol: string,
+    hubPoolTokenAddress: string,
```

Comment:
```suggestion
    hubPoolToken: L1Token,
```
 (Plus would need to remove the `symbol` argument above)

Reply:
Forget this - seems easier for me to just adjust it afterwards :+1: